README.md

```diff
@@ -30,7 +30,10 @@ Using
 =========
 
 Make sure you have the SPARK_HOME enviroment variable set correctly, as
-SparklingPandas uses this for including the PySpark libraries
+SparklingPandas uses this for including the PySpark libraries.
+
+Some SparklingPandas components are built on the JVM, if building from source you
+can compile these run `./sbt/sbt assembly`.
```

Comment:
So this is kind of tricky to avoid since we depend on using spark-submit & friends (also Spark doesn't really install its python code very nicely).